pkgs/os-specific/linux/fwts/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchzip, autoreconfHook, pkgconfig, glib, libtool, pcre
+, json_c, flex, bison, dtc, pciutils, dmidecode, iasl }:
+
+stdenv.mkDerivation rec {
+  name = "fwts-${version}";
+  version = "18.01.00";
+
+  src = fetchzip {
+    url = "http://fwts.ubuntu.com/release/fwts-V18.01.00.tar.gz";
```

Comment:
Consider reusing `version` here, to simplify bumps

Reply:
Fixed, thanks.